tftrt/examples/image-classification/image_classification.py

```diff
@@ -553,6 +554,12 @@ def get_frozen_graph(
     num_nodes['native_tf'] = len(frozen_graph.node)
     graph_sizes['native_tf'] = len(frozen_graph.SerializeToString())
 
+    # Cache graph before trt conversion for reference
```

Comment:
What happens if we also cache when `--use_trt=False`?

Reply:
User can always run the example with `use_trt=False` if they want to write the native TF graph in cache.

Reply:
that's true; the only convenience of this is that when running with use_trt=true you can get both graphs in the same run.

Reply:
Actually I see, with use_trt false you get two graphs, not very clean solution....; I'll remove this part, the important piece of this PR is the trt engine anyway.